design/subscriber-sync.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Newsletter Subscribers synchronization
+  - As a store owner
+  - I want to update subscribers automatically
```

Comment:
Maybe "add" or "import" is a better word there as Smaily uses **add subscribers** in its own UI.

![image](https://user-images.githubusercontent.com/22918359/71715657-d2ddcc80-2e1a-11ea-94e3-ce5fc84f8f9a.png)

Reply:
On the other hand, this feature is not only for importing it should be something like 

>As a store owner
I want to automatically keep subscribers subscription status up to date
So that I will save time on my newsletter preparations

Reply:
Yes, that is much better, but "keep subscribers subscription status" would mean that the extension is only synchronizing subscription status, but not additional fields. Come to think of it. Maybe it is OK, that `subscriber-sync.feature` only describes subscription status synchronization and `additional-data.feature` handle all of the other fields.

Reply:
Yup, that was the idea. Separate the subscription status and additional fields. This also comes out from the benefit part of the feature description.

Reply:
Changed in https://github.com/sendsmaily/Sendsmaily-Sync-for-Magento/pull/8/commits/b7a46ad32f918c2cdb2c5284bee4ab32568ce96f